doc/CMakeLists.txt

```diff
@@ -141,8 +141,14 @@ set(CELERITAS_USE_BREATHE ${CELERITAS_USE_Breathe})
 set(CELERITAS_USE_SPHINXBIBTEX ${CELERITAS_USE_SphinxBibtex})
 set(CELERITAS_USE_SPHINXRTDTHEME ${CELERITAS_USE_SphinxRtdTheme})
 set(Celeritas_SHORT_VERSION ${Celeritas_VERSION})
+message("Celeritas_SHORT_VERSION: ${Celeritas_SHORT_VERSION}")
+message("Celeritas_VERSION_STRING: ${Celeritas_VERSION_STRING}")
```

Comment:
Was this for debugging, or did you want to keep it in?

Reply:
Debugging. Thanks for the catch!